tests/tests/t01-inloggen-basisgebruiker.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { expect, test } from '@playwright/test';
+
+import { goToPageAndAcceptCookies } from '../helpers/go-to-page-and-accept-cookies';
+import { loginOnderwijsHetArchief } from '../helpers/login-onderwijs-het-archief';
+
+/**
+ * New: https://docs.google.com/spreadsheets/d/1IvhK0v0HSntCwTcXiFseHargwwWwpoCkDMjmMehaDMA/edit#gid=0
+ */
+
+test('T01: Inloggen basisgebruiker', async ({ page, context }) => {
+	await goToPageAndAcceptCookies(
+		page,
+		process.env.TEST_CLIENT_ENDPOINT as string,
+		// INT
+		//'Audiovisuele beeldbank voor leerkrachten | Het Archief voor Onderwijs'
```

Comment:
you should be able to fix this by updating the translations in the INT database and then restarting the INT proxy server